src/sections/OurDoctors/StarRating.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import "@fortawesome/fontawesome-free/css/all.css";
+
+const StarRating = ({ rating }) => {
```

Comment:
Shift this to `component` folder and also add a storybook for this.

Reply:
Resolved. thanks!